README.md

```diff
@@ -74,6 +74,18 @@ FOR A PARTICULAR PURPOSE. See the GNU General Public License for more details.
    code in Cacti that is non-compliant and it should be fixed up wherever
    possible.
 
+5. [Plugins](README.md#cacti-plugins)
+
+   This section contains all Plugin related information.
+   Guidelines, hooks, references, etc.
+   More How To's can also be found at [Cacti's forum](https://forums.cacti.net/viewforum.php?f=6)
+
+6. How To's
```

Comment:
Need to put link in to the sub-topic.  It is always the heading converted to lowercase, with spaces replaced by hyphens.

Reply:
Yep, saw that one. Got it ready.